src/shared/config/db.ts

```
@@ -0,0 +1,596 @@
+  down?: string;
+}
+
+export interface HealthCheckResult {
+  connected: boolean;
+  latency?: number;
+  error?: string;
+  timestamp: Date;
+}
+
+export class Database {
+  private pool: postgres.Sql<{}> | null = null;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)